Retrovizor.Domain/Repositories/Implementations/StudentClassRepository.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Retrovizor.Data.Entities;
+using Retrovizor.Data.Entities.Models;
+using Retrovizor.Domain.Repositories.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Retrovizor.Domain.Repositories.Implementations
+{
+    public class StudentClassRepository : IStudentClassRepository
```

Comment:
Nisam siguran jesu li repository-i za medutablice uopce potrebni

Reply:
Mislin da ce za neke tribat jer imaju npr. DateTime neki koji bi tribalo nekad editat i slicno pa san ih nabrzinu otipka just to stay safe, lako ih izbrisemo posli.